src/shared/constants/balance-questions.cat-a-mod2.constants.ts

```diff
@@ -0,0 +1,10 @@
+/* tslint:disable */
+import { VehicleChecksQuestion } from '../../providers/question/vehicle-checks-question.model';
+
+// TODO - PREP-AMOD2: Add balance questions
+export default [
+
```

Comment:
is there a further ticket addressing this `todo` or should it be implemented now?

Reply:
MES_4670 deals with this. Will add it to the comment too